certified-connectors/Azure Machine Learning/apiDefinition.swagger.json

```diff
@@ -0,0 +1,16597 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "title": "Azure Machine Learning",
+        "description": "Azure Machine Learning REST CLI (v2)",
```

Comment:
@yashodeepm please make sure that description reflects the main value and purpose offered by the connector. More information about the description can be found [here](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#write-a-description-for-your-connector).

Reply:
Changes made